shared/chat/conversation/container.js

```diff
@@ -52,6 +52,12 @@ class ConversationContainer extends Component<void, Props, State> {
     this.setState({listScrollDownCounter: this.state.listScrollDownCounter + 1})
   }
 
+  // We wrap this so children don't churn when this.props.onBack() changes due to this component churning. Whe this thing does less we can
+  // likely remove this
+  _onBack = () => {
```

Comment:
This component currently churns a bunch which means the caching doesn't work well and therefore the dispatch mapping gets called a bunch which causes this.props.onBack to mutate, and since we pass that as an ownProps to Header that was causing re-render. When this thing is better about its own cache we can remove this.